packages/wallet/src/components/CredentialDetail.tsx

```diff
@@ -162,16 +127,10 @@ const CredentialDetail: NavigationElement = ({ navigation, route }) => {
                 {display?.description}
               </Section>
             </View>
-
-            {properties}
           </View>
         )}
 
-        {approvalDate && <Section title="Approved">{approvalDate}</Section>}
-
-        {score && <Section title="Score">{score}</Section>}
-        {scoreType && <Section title="Score Type">{scoreType}</Section>}
-        {provider && <Section title="Provider">{provider}</Section>}
+        {properties}
```

Comment:
Originally, the properties were hidden on the screen because `header`'s `flexDirection` was "row". Pulled them out of `header` and placed in the same level as `revoked` and `expirationDate`.